libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -91,6 +91,17 @@ class Strings {
         assertPrints(proteins.take(5).toList(), "[Isoleucine, Arginine, Glycine, Arginine, Glutamine]")
     }
 
+    @Sample
+    fun elementAt() {
+        val string = "kotlin"
+        assertPrints(string.elementAt(0), "k")
+        assertPrints(string.elementAt(5), "n")
+        assertFailsWith<StringIndexOutOfBoundsException> { string.elementAt(6) }
```

Comment:
`StringIndexOutOfBoundsException` is not an exception type that is guaranteed to be thrown on every platform. Use just `IndexOutOfBoundsException`.

Reply:
Thanks for your review.
I didn't think of every platform.
I will fix it.